test/__snapshots__/PoolAddress.spec.ts.snap

```diff
@@ -1,5 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`PoolAddress #computeAddress gas cost 1`] = `642`;
+exports[`PoolAddress #computeAddress gas cost 1`] = `643`;
 
-exports[`PoolAddress #computeAddress matches example from core repo 1`] = `"0x03D8bab195A5BC23d249693F53dfA0e358F2650D"`;
+exports[`PoolAddress #computeAddress matches example from core repo 1`] = `"0xcf75931a4C18d87F784a91719DefFDdb74b7B0d5"`;
```

Comment:
Don't think this should change?

Reply:
The initcode hash changed for the 0.8 migration in core, so the create2 addresses change

Reply:
Added some logs to core-v3#0.8 to test:
```
bytecodeHash 0xa598dd2fba360510c5a8f02f44423a4468e902df5857dbce3ca162a43a3a31ff
factory 0x5FbDB2315678afecb367f032d93F642f64180aa3
tokens [
  '0x1000000000000000000000000000000000000000',
  '0x2000000000000000000000000000000000000000'
]
feeAmount 250
bytecodeHash 250
create2Address 0xcf75931a4C18d87F784a91719DefFDdb74b7B0d5
      ✔ enables pool creation (126ms)

```